media_softlet/linux/Xe_M_plus/ddi/media_libva_caps_mtl_base.cpp

```diff
@@ -204,7 +204,7 @@ VAStatus MediaLibvaCapsMtlBase::LoadAv1EncProfileEntrypoints()
         attribValAV1ToolsExt2.bits.tile_size_bytes_minus1 = 3;
         attribValAV1ToolsExt2.bits.obu_size_bytes_minus1  = 3;
         attribValAV1ToolsExt2.bits.max_tile_num_minus1    = 511;
-        attribValAV1ToolsExt2.bits.tx_mode_support        = 2;
+        attribValAV1ToolsExt2.bits.tx_mode_support        = 4;
```

Comment:
For MTL, this file is not in use. Caps will be read from a caps table file. Could you help to change this file at the same time? Thanks. https://github.com/intel/media-driver/blob/master/media_softlet/linux/xe_lpm_plus_r0/encode/av1/ddi/capstable_data_av1_encode_xe_lpm_plus_r0_specific.h#L37

Reply:
Resolved, thanks.